### PR TITLE
community: smoother reports export (fixes #7670)

### LINKED ISSUE #7670: Export to CSV option shown when nothing is available to export in reports
csverror

export to csv option is shown even when there is nothing to export

### PR DESCRIPTION
**Pull request by Gavinp14**
reportsissuesolved

export to csv button only appears if there are reports available now

### CODE REVIEW

File: src/app/teams/teams-reports.component.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="action-buttons" *ngIf="editable">
   <button mat-raised-button color="primary" i18n (click)="openAddReportDialog()">Add Report</button>
-  <button mat-raised-button color="primary" i18n (click)="exportReports()">Export as CSV</button>
+  <button mat-raised-button color="accent" i18n *ngIf="reports && reports.length > 0" (click)="exportAsCSV()">
```
Comment (RheuX) [inline src/app/teams/teams-reports.component.html:3]: Wondering why you change the click function into a `exportAsCSV()`, export now doesnt work, as it does nothing.

Screenshot 2024-10-28 at 11 51 02 AM

please change it back to `exportReports()`

**Review by RheuX — COMMENTED** [review]
there are some spaces that was entered in some lines, thats not relevant to the code, just wondering if you did it intentionally to be readable?

but please check my comment below; for the main issue

File: src/app/teams/teams-reports.component.html (hunk 1 above)
Comment (Gavinp14) [reply src/app/teams/teams-reports.component.html:3]: Also, I think this is some formatting errors that my IDE is doing. I'll go back and change these as well...

Comment (RheuX) [reply src/app/teams/teams-reports.component.html:3]: sounds good!

**Review by Mutugiii — APPROVED** [review]

**Review by RheuX — APPROVED** [review]
lgtm

**Review by dogi — APPROVED** [review]

**Pull request merged**